src/main/java/com/conveyal/r5/analyst/TravelTimeComputer.java

```diff
@@ -280,6 +282,7 @@ public void computeTravelTimes(OutputStream os) throws IOException {
 
                         }
 
+                        // Record the paths used
```

Comment:
Even the word "paths" is pretty vague unless you're already in the mindset of working on this particular problem. I'd prefer something like "Record the sequence of transit trips and stops used to reach the destination (the paths)."